onedal/tests/utils/__init__.py

```diff
@@ -0,0 +1,51 @@
+# ===============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import numpy as np
+
+from ._dataframes_support import get_dataframes_and_queues
+from ._device_selection import (
+    get_memory_usm,
+    get_queues,
+    is_dpctl_device_available,
+    pass_if_not_implemented_for_gpu,
+)
+
+basicstats_options_and_tests = {
```

Comment:
Is it possible to place this definition into `basic_statistics/tests` module?
Because `utils` does not seem to be a proper place for this kind of definitions.

I like the overall basic statistics tests refactoring, it's just the placement of this dictionary looks not so good to me.

Reply:
I'll move it an add another omission to the coverage.rc file for that case, otherwise it will be picked up in coverage metrics.